### ISSUE TITLE
Required file missing for entering development shell at v1.0.0

### ISSUE THREAD

**Issue by bwbush** [input-output-hk/partner-chains#45]
**Describe the bug**

Entering a nix development shell fails for v1.0.0:

```console
$ nix develop
. . .
       error: unable to download 'https://github.com/input-output-hk/trustless-sidechain/archive/04f656d4e755dbc4f5ce384957c2859695d7147c.tar.gz': HTTP error 404

       response body:

       Not Found
```

**To Reproduce**

```console
$ git clone git@github.com:input-output-hk/partner-chains
$ cd partner-chains
$ git checkout v1.0.0
$ nix develop
```

**Expected behavior**
This missing file should be available.

**Environment**

```console
$ nixos-version
24.05.4169.797f7dc49e0b (Uakari)

$ nix --version
nix (Nix) 2.18.5

$ git latest      
52fedddbaa65d045ee23a8ccedac113831097aa7 (HEAD, tag: v1.0.0, origin/v1.0) Add note to README (#5)
```

**Comment by tgunnoe** [comment]
Hello, thanks for checking this out!  In the process of open sourcing our repositories and drafting releases, we had some leftover renaming not corrected and the devshells were affected by that. They have been since fixed for both master and the latest v1.1.0 release. Sorry for the inconvenience.

**Comment by bwbush** [comment]
Thanks for the update. I see the full v1.1.0 is available now, so I'll switch over to that.
